server/src/optima/project.py

```diff
@@ -615,7 +615,15 @@ def uploadExcel():
         D.G.workbookname = D.G.projectname + '.xlsx'
         D.G.inputprograms = deepcopy(project.programs)
         D.G.inputpopulations = deepcopy(project.populations)
-        D = updatedata(D, input_programs = project.programs, savetofile = False)
+
+        # Is this the first time? if so then we have to run simulations
+        try:
+            D.S # the cool kids don't reRun when they already have simulations
+            shouldReRun = False
+        except AttributeError:
+            shouldReRun = True
```

Comment:
@sebastianconcept hehe, yeah I read some underscore vs camelCase & camelCase seems to be better. Indeed in python underscore is the default & all libs we are using underscore.

If you would start out with a fresh python project would you go for camelCase for the beauty or underscore to have consistent python codebase?